front/app/Components/AreaIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Image from "next/image"; // Import de next/image
+
+interface AreaIconProps {
```

Comment:
Create an interface file for that in the folder interfaces

Reply:
Je le fait dans quel dossier ?